packages/venia-concept/src/components/App/app.js

```diff
@@ -69,17 +72,7 @@ const App = props => {
                 addToast(errorToastProps);
             }
         }
-    }, [errors, handleDismiss]);
-
-    if (renderError) {
-        return (
-            <Fragment>
-                <Main isMasked={true} />
-                <Mask isActive={true} />
-                <ToastContainer />
-            </Fragment>
-        );
-    }
+    }, [errors]); // eslint-disable-line
```

Comment:
This and the other eslint-disable should be addressed by #1377

Reply:
I think you accidentally removed `handleDismiss` from the array; re-add and this is ready to QA.